core/domain/services/graphql/connection_resolvers/AbstractConnectionResolver.php

```diff
@@ -34,6 +35,25 @@ protected function getLimit()
         return $limit;
     }
 
+    /**
+     * Determine whether or not the the offset is valid, i.e the entity corresponding to the
+     * offset exists. Offset is equivalent to entity ID. So this function is equivalent to
+     * checking if the entity with the given ID exists.
+     *
+     * @access public
+     *
+     * @param int $offset The ID of the node used for the cursor offset
+     *
+     * @return bool
+     */
+    // phpcs:ignore PSR1.Methods.CamelCapsMethodName.NotCamelCaps
+    public function is_valid_offset($offset)
+    {
+        $entity = $this->get_query()->get_one_by_ID($offset);
```

Comment:
googled this and got results to our docs :)

![image](https://user-images.githubusercontent.com/1477453/76949066-ae20cd80-6910-11ea-9b95-e6ab3304c4fd.png)